lib/synced/model.rb

```diff
@@ -142,7 +141,7 @@ def synchronize(scope: scope_from_relation, strategy: synced_strategy, **options
         tolerance:             synced_tolerance,
         synced_endpoint:       synced_endpoint
       })
-      Synced::Synchronizer.new(self, options).perform
+      Synced::Synchronizer.new(self, strategy: strategy, **options).perform
```

Comment:
Just ** unsplating options would work as well? But I understand that having strategy as an explicit key matches the interface more directly.
```suggestion
      Synced::Synchronizer.new(self, **options).perform
```

Reply:
correct